siege_utilities/geo/spatial_data.py

```
@@ -343,20 +343,67 @@ def _dirs_from_inventory(year: int, inventory: Optional[dict] = None) -> Optiona
-    ``TIGER{year}/`` directory listing.  These subdirectories exist in every
-    annual TIGER release from 2010 onward.  The generic ``CD`` directory is
-    included because Census uses it for both national pre-2022 files
-    (``tl_{year}_us_cd{congress}.zip``) and per-state 2022+ files
-    (``tl_{year}_{fips}_cd{congress}.zip``).
+    Used when the Census FTP returns 429 (rate-limit) or 403 (directory
+    listing blocked) on the top-level ``TIGER{year}/`` directory.  Returning
+    a *known-good* list lets callers proceed with construct-then-download
+    instead of failing the whole job because index.html is unreachable.
+
+    The list is conservative — every directory enumerated here has been
+    confirmed present in TIGER releases 2014 onward.  Earlier years (2010–
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)